### ISSUE TITLE
Error for Windows 10

### ISSUE THREAD

**Issue by koticik** [chrome-php/headless-chromium-php#202]
Installed xampp on the computer installed composer
  If I execute this code
```
<?php
require __DIR__ . '/vendor/autoload.php';
use HeadlessChromium\BrowserFactory;
$factory = new BrowserFactory('C:\Program Files (x86)\Google\\Chrome\Application\chrome.exe');
$browser = $factory->createBrowser();
    // creates a new page and navigate to an url
    $page = $browser->createPage();
    $page->navigate('http://google.ru/')->waitForNavigation();

    // get page title
    $pageTitle = $page->evaluate('document.title')->getReturnValue();

    // screenshot - Say "Cheese"!
    $page->screenshot()->saveToFile('bar.png');

    // pdf
    $page->pdf(['printBackground' => false])->saveToFile('bar.pdf');
    $browser->close();
?>
```
`Fatal error: Uncaught HeadlessChromium\Exception\OperationTimedOut: Operation timed out (5sec) in K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\Utils.php:69 Stack trace: #0 K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\Communication\ResponseReader.php(108): HeadlessChromium\Utils::tryWithTimeout(5000000, Object(Generator)) #1 K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\Communication\Session.php(84): HeadlessChromium\Communication\ResponseReader->waitForResponse(5000) #2 K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\Browser.php(170): HeadlessChromium\Communication\Session->sendMessageSync(Object(HeadlessChromium\Communication\Message)) #3 K:\xampp\htdocs\test\test.php(39): HeadlessChromium\Browser->createPage() #4 {main} thrown in K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\Utils.php on line 69`

Found this option in responses to other users
```
  $browser = $factory->createBrowser([
         'debugLogger' => 'php://stdout',
      'sendSyncDefaultTimeout' => 5000
]);
```
As a result, my browser starts with an empty window, hangs for a while and closes, after which I get this error.
`Fatal error: Uncaught Symfony\Component\Process\Exception\ProcessTimedOutException: The process ""C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --version" exceeded the timeout of 60 seconds. in K:\xampp\htdocs\test\vendor\symfony\process\Process.php:1213 Stack trace: #0 K:\xampp\htdocs\test\vendor\symfony\process\Process.php(425): Symfony\Component\Process\Process->checkTimeout() #1 K:\xampp\htdocs\test\vendor\symfony\process\Process.php(249): Symfony\Component\Process\Process->wait() #2 K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\BrowserFactory.php(86): Symfony\Component\Process\Process->run() #3 K:\xampp\htdocs\test\vendor\chrome-php\chrome\src\BrowserFactory.php(60): HeadlessChromium\BrowserFactory->getChromeVersion() #4 K:\xampp\htdocs\test\test.php(32): HeadlessChromium\BrowserFactory->createBrowser(Array) #5 {main} thrown in K:\xampp\htdocs\test\vendor\symfony\process\Process.php on line 1213`
Tell me how you can fix it?

**Comment by canuk-1** [comment]
I had the same issue - change the timeout to something bigger (I started at 50000 and reduced).

**Comment by maximgrynykha** [comment]
@koticik I was tested your issue with another environment [Windows 10 x64 | OpenServer | PHP 7.4] and its works fine. Got files (.png & .pdf) by code as you listed above. First of all, I don't actually know where you have a chrome.exe but try to check & change the path to it in the next - "C:\Program Files\Google\Chrome\Application\chrome.exe" way. Secondly, I'll recommend you to use [OpenServer](https://ospanel.io/). If all of it will not work try [recommendation by @canuk-1](https://github.com/chrome-php/headless-chromium-php/issues/202#issuecomment-814430293)

**Comment by AyanBhadury** [comment]
@ismaxim Getting the same issue is it not compatible with windows? 

```
<?php

require './vendor/autoload.php';

use HeadlessChromium\BrowserFactory;

$browserFactory = new BrowserFactory();

// starts headless chrome
$browser = $browserFactory->createBrowser([
    'windowSize' => [860, 680],
   'debugLogger' => 'php://stdout',
    'ignoreCertificateErrors'	=> true,
    'sendSyncDefaultTimeout' =>7000
]);

try {
    // creates a new page and navigate to an url
    $page = $browser->createPage();
    $page->navigate('https://github.com/')->waitForNavigation();

    // get page title
    $pageTitle = $page->evaluate('document.title')->getReturnValue();

    // screenshot - Say "Cheese"! 😄
    $page->screenshot()->saveToFile('./foo/bar.png');

    // pdf
    $page->pdf(['printBackground' => false])->saveToFile('./foo/bar.pdf');
} finally {
    // bye
    $browser->close();
}
```

**Comment by AyanBhadury** [comment]
![image](https://user-images.githubusercontent.com/17782121/122869811-69bd8200-d34a-11eb-90f2-3fcda9c17bdd.png)

**Comment by enricodias** [comment]
If you are using debug logger on windows, see #227

**Comment by GrahamCampbell** [comment]
Fixed in v0.11.0. :)
